### ISSUE TITLE
Option to use fixed angle step in Rainbow reader

### ISSUE THREAD

**Issue by egouden** [openradar/xradar#75]
In Rainbow, the radar measurements can be sampled for a given fixed angle step (fixselect>Anglestep)

It would be nice to have the option to use this as regular nominal coordinates instead of the actual azimuth angles.

**Comment by kmuehlbauer** [comment]
That would be specific to that backend, but we might have the chance to use `reindex_angle` for that. I'll let this sink in a bit.

**Comment by egouden** [comment]
I think reading the nominal angles should be the default for all readers for several reasons:

- the actual physical quantity is an azimuth average of about 30 radar beams
- most applications want the nominal angles
- concatenation is easier

 The actual angles (due to imperfection of the radar system) could be read as optional metadata or actual coordinates as another option.

What do you think?

**Comment by mgrover1** [comment]
Yeah - I agree that setting some reasonable defaults/adding a bit more logic on attempting to calculate some of the fields required would be great! 

This is a similar practice to reading in terabytes of climate model output where the exact latitude/longitude can vary a bit. When using `open_mfdataset`, the file uses the first or last file to use as the coords and aligns everything to that. This is not necessarily what we **need** to do/follow here, but a similar practice.

**Comment by kmuehlbauer** [comment]
With the new xarray indexers https://xarray-indexes.readthedocs.io/index.html we might be able to create azimuth/elevation/time indexes as well as range indexes which are lazy and conform to @egoudens suggestion to use nominal angles.
